.eslintrc.js

```diff
@@ -5,6 +5,8 @@ module.exports = {
         'plugin:@typescript-eslint/recommended',
         'prettier/@typescript-eslint',
         'plugin:prettier/recommended',
+        // For disabling react/react-in-jsx-scope
+        'plugin:react/jsx-runtime',
```

Comment:
`deriv-app` still uses react 16 and we don't bundle react along with `@deriv/ui` package. So, we should not include this plugin yet - `plugin:react/jsx-runtime`.

Reply:
We expect `react` to be a peer dependency. When we use this package in `deriv-app`, it would pick react16 instead.

Reply:
I think it's fixed here - https://github.com/binary-com/deriv-components/pull/261/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R86

Reply:
Sorted.